README.md

```diff
@@ -144,6 +144,76 @@ It appears from the benchmarks that:
 For now, I'm quite happy with these benchmarks and hope we can make them even
 better in the future.
 
+## Sample Poolboy usage
+
+If you are a beginner and want to use this lib in your project it is maybe not
+apparent how you use it. This is a small walkthrough on how to use the lib
+with a supervisor to get connection pooling wrapped in a public API.
+
+To start with you need to make a module that implements `Supervisor`.
+```
+defmodule Redix.Poolboy do
+  use Supervisor
+
+  @connection_params Application.get_env(:redix, :poolboy)[:server]
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  def init([]) do
+    worker_pool_options = [
+      name: {:local, :redix_poolboy},
+      worker_module: Redix, #worker process
+      size: 10,
+      max_overflow: 5
+    ]
+
+    children = [
+      :poolboy.child_spec(:redix_poolboy, worker_pool_options, @connection_params)
+    ]
+
+    opts = [strategy: :one_for_one, name: __MODULE__]
+
+    supervise(children, opts)
+  end
+
+
+  def command(command) do
+    :poolboy.transaction(:redix_poolboy, fn(conn) -> Redix.command(conn, command) end)
+  end
+
+  def pipeline(command) do
+      :poolboy.transaction(:redix_poolboy, fn(conn) -> Redix.pipeline(conn, command) end)
```

Comment:
I have no idea what happened in the copy paste, it is indented like it should be here in my editor.

If you dont mind fixing yourself that might be best, I squeezed in some time at work today to get this pull request out the door.. hectic days rewriting one of our apps to elixir :)